README.md

```diff
@@ -99,6 +99,30 @@ NAME      AGE
 example   15s
 ```
 
+After a couple of minutes, all cluster members should become ready.
+
+```
+$ kubectl get zookeepercluster
+
+NAME      REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT    EXTERNAL ENDPOINT   AGE
+example   3          3                latest                      10.100.200.18:2181   N/A                 94s
+```
+Note: when the Version field is set as well as Ready Replicas are equal to Replicas that signifies our cluster is in Ready state
+
+Additionally, check the output of describe command which should show the following cluster condition:-
+
+```
+$ kubectl describe zookeepercluster
+
+Conditions:
+  Last Transition Time:    2020-05-18T10:17:03Z
+  Last Update Time:        2020-05-18T10:17:03Z
+  Status:                  True
+  Type:                    PodsReady
+
+```
+Note: User should wait for the Pods Ready condition to be True
```

Comment:
Could u add > in front of the Note to keep the formatting consistent?

Reply:
I have added this.